workflow-testing/src/main/java/com/squareup/workflow1/testing/WorkerTester.kt

```diff
@@ -31,7 +39,7 @@ public interface WorkerTester<T> {
   /**
    * Suspends until the worker emits an output or finishes.
    *
-   * Throws an [AssertionError] if an output was emitted.
+   * Throws an [AssertionError] if an output was emitted or the Worker has an error.
```

Comment:
Prior to this change, we would not throw if there was an exception (we counted that as 'finished'). I changed that, in  favour of explicitly requiring the Worker to complete unexceptionally as a Turbine would.

This seemed the best choice to me. WDYT @RBusarow @rjrjr ?

Reply:
Yep, I agree.